fe/fe-core/src/main/java/com/starrocks/catalog/CatalogUtils.java

```diff
@@ -235,4 +236,16 @@ public static int calBucketNumAccordingToBackends() {
         }
         return bucketNum;
     }
+
+    public static void maskProperties(Map<String, String> properties) {
+        // aws.s3.access_key -> ******
+        properties.computeIfPresent(CloudConfigurationConstants.AWS_S3_ACCESS_KEY, (k, v) -> "******");
+        // aws.s3.secret_key -> ******
+        properties.computeIfPresent(CloudConfigurationConstants.AWS_S3_SECRET_KEY, (k, v) -> "******");
```

Comment:
I recommend masking part of characters, it's better for a user to check credentials.
For example, original `ak=efi923#31` => `ak=ef**31`.

Reply:
> I recommend masking part of characters, it's better for a user to check credentials. For example, original `ak=efi923#31` => `ak=ef**31`.

done